helm-chart/csi-driver/values.yaml

```diff
@@ -0,0 +1,38 @@
+# Secrets:
+# apiToken [Required] - Must be a Linode APIv4 Personal Access Token with all permissions. (https://cloud.linode.com/profile/tokens) 
+apiToken: ""
+
+# region [Required] - Must be a Linode region. (https://api.linode.com/v4/regions)
+region: ""
+
+# Default namespace is "kube-system" but it can be set to another namespace
+namespace: kube-system
+
+# Images - Default 
```

Comment:
Lets try to use registry.k8s.io instead of k8s.gcr.io if possible.
https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/

Reply:
Didn't know they switched registry. I'll test the new registry and update it!